privacyidea/lib/tokens/registrationtoken.py

```diff
@@ -164,7 +165,17 @@ def update(self, param):
             # Otherwise genkey and otpkey will raise an exception in
             # PasswordTokenClass
             del param["genkey"]
-        param["otpkey"] = generate_password(size=self.otp_len)
+        if "registrationcode_length" in param:
```

Comment:
We tried to to have a token specific parameter naming like ``<type>.<parameter>`` to avoid name clashes.
I recommend changing the parameter ot ``registration.length`` and ``registration.contents`` respectively.

Reply:
Changed.